corehq/apps/hqwebapp/migrations/0009_truncate_authtoken_table.py

```diff
@@ -0,0 +1,17 @@
+from django.db import migrations
+
+from corehq.util.django_migrations import add_if_not_exists_raw
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0008_hqoauthapplication'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            add_if_not_exists_raw("TRUNCATE TABLE authtoken_token", name='authtoken_table'),
```

Comment:
I think the logic in this needs to be inverted for this use case. As it stands it will execute this SQL if the table does not exist.

Reply:
🤦🏻 of course! Just added an inverse utility and tested the logic in a psql shell which worked.